common/game_rules/zz_hfe_vanilla_replace_game_rules.txt

```diff
@@ -112,23 +112,18 @@ can_spawn_debris = {
 		or = {
 			is_country_type = default
 			is_country_type = hfe_pirate
```

Comment:
Also im not sure if pirates need debris, maybe better to remove this vanilla overwriting

Reply:
Okay, done